Ginger/GingerCore/ALM/QtestCore.cs

```diff
@@ -0,0 +1,657 @@
+                return false;
+            }
+
+
+            return true;
+        }
+
+        public bool ExportBusinessFlowToALM(BusinessFlow businessFlow, QtestTestSuite mappedTestSuite, string parentObjectId, ObservableList<ExternalItemFieldBase> testSetFields, ref string result)
+        {
+            ConnectALMServer();
+            testrunApi = new QTestApi.TestrunApi(connObj.Configuration); 
+            testcaseApi = new QTestApi.TestcaseApi(connObj.Configuration);
+            testsuiteApi = new QTestApi.TestsuiteApi(connObj.Configuration);
+
+            ObservableList<ActivitiesGroup> existingActivitiesGroups = new ObservableList<ActivitiesGroup>();
+            try
+            {
+                QTestApiModel.TestSuiteWithCustomFieldResource testSuite = null;
+                if (mappedTestSuite == null)
+                {
+                    testSuite = new QTestApiModel.TestSuiteWithCustomFieldResource( null, null, null, null,
+                                                                                    businessFlow.Name,
+                                                                                    new List<QTestApiModel.PropertyResource>());                   
+                    testSuite = testsuiteApi.CreateTestSuite((long)Convert.ToInt32(ALMCore.DefaultAlmConfig.ALMProjectKey), testSuite, (long)Convert.ToInt32(parentObjectId), "test-cycle");
+
+                    foreach (ActivitiesGroup ag in businessFlow.ActivitiesGroups)
+                    {
+                        QTestApiModel.TestRunWithCustomFieldResource testRun = new QTestApiModel.TestRunWithCustomFieldResource(null, null, null, null,
+                                                                                                                                ag.Name,
+                                                                                                                                new List<QTestApiModel.PropertyResource>(),
+                                                                                                                                testcaseApi.GetTestCase((long)Convert.ToInt32(ALMCore.DefaultAlmConfig.ALMProjectKey), (long)Convert.ToInt32(ag.ExternalID))); 
+                        testRun = testrunApi.Create((long)Convert.ToInt32(ALMCore.DefaultAlmConfig.ALMProjectKey), testRun, testSuite.Id, "test-suite");                      
+                    }     
+                }
+                else
+                {
+                    //##update existing TestSuite
+                    QTestApiModel.TestSuiteWithCustomFieldResource testSuiteToUpdate = testsuiteApi.GetTestSuite((long)Convert.ToInt32(ALMCore.DefaultAlmConfig.ALMProjectKey), (long)Convert.ToInt32(mappedTestSuite.ID));
+
+                    //foreach (QtestTest test in mappedTestSuite.Tests)
+                    //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)